Answers.md

```diff
@@ -0,0 +1,31 @@
+## Self-Study Questions
+
+Demonstrate your understanding of this week's concepts by answering the following free-form questions.
+
+Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
+
+1. Describe the biggest difference between `.forEach` & `.map`.
+
+.forEach make changes to the original array.
+.map won't change the original array but create a new one.
+
+2. What is the difference between a function and a method?
+
+Functions are block of code that can be call whenever desire.
+Method is also a function but it has been hosted inside and Object and given a key name mostly.
+
+3. What is closure?
+
+A Closure is the combined operation of a function with his environment.
```

Comment:
This is correct, but more technically, a closure is a nested function that has access to it's outter function's variables.  Closures store references to the outter functions variables even after function invocation has completed.